lib/v2/dhu/xxgk/news.js

```diff
@@ -0,0 +1,31 @@
+const got = require('@/utils/got');
+const iconv = require('iconv-lite');
+const cheerio = require('cheerio');
+const resolve_url = require('url').resolve;
+
+const base_url = 'https://xxgk.dhu.edu.cn/1737/list.htm';
+
+module.exports = async (ctx) => {
+    const link = `${base_url}`;
```

Comment:
```suggestion
    const link = base_url;
```
Please fix lint problem